chainer/function_node.py

```diff
@@ -146,7 +147,11 @@ class FunctionNode(object):
     _retained_output_data = None
     _local_function_hooks = None
     _supports_static_optimizations = False
-    _is_chainerx = None
+    # True if the function node is operating on ChainerX arrays and it falls
+    # back to NumPy/CuPy implementation.
+    _is_chainerx_fallback = False
```

Comment:
In this implementation, `False` will be set to this variable whenever `is_chainerx` is `False` (even if chainerx implementation of the function is not supported).  I think it seems a little bit confusing.  How about assign `None` or anything like that which means "unknown state" in this line (or delete this line)?

Reply:
Renamed to `_is_chainerx_fallback_mode` as discussed offline